README.md

```diff
@@ -21,20 +63,19 @@ Their values are taken as input as environment variables/ configuration for conn
 Various account data provided by Adyen are necessary to be configured so that the requests from the connect application can be authenticated by Adyen platform within the integration.
 Their values are taken as input as environment variables/ configuration for connect with variable names `ADYEN_API_KEY`, `ADYEN_NOTIFICATION_HMAC_KEY`, `ADYEN_MERCHANT_ACCOUNT`, `ADYEN_CLIENT_KEY`, `ADYEN_LIVE_URL_PREFIX` and `ADYEN_ENVIRONMENT`.
 
-## Getting started
-The template contains two modules :  
-- Enabler: Acts as a wrapper implementation in which frontend components from Adyen embedded. It gives control to checkout product on when and how to load the connector frontend based on business configuration. In cases connector is used directly and not through Checkout product, the connector library can be loaded directly on frontend than the PSP one.
-- Processor : Acts as backend services which is middleware to integrate with Adyen platform. It is mainly responsible for managing transactions with Adyen and updating payment entity in composable commerce.  `connect-payment-sdk` will be offered to be used in connector to manage request context, sessions and other tools necessary to transact.
+## Development Guide
+Regarding the development of enabler module, please refer to the following documentations:
+- [Development of Enabler](./enabler/README.md)
 
 Regarding the development of processor module, please refer to the following documentations:
 - [Development of Processor](./processor/README.md)
 
-#### 1. Register as connector in commercetools Connect
+#### Register as connector in commercetools Connect
 Follow guidelines [here](https://docs.commercetools.com/connect/getting-started) to register the connector for public/private use.
```

Comment:
Can this changed to 
"Use public connector listed in connect marketplace. If any customization done, follow guidelines [here](https://docs.commercetools.com/connect/getting-started) to register the connector for private use."

and title to "Connector in commercetools Connect"

Reply:
https://github.com/commercetools/connect-payment-integration-adyen/pull/76/commits/beedf49b43c98849b70101b1995e56aab1514931

Reply:
@prateek-ct missed this issue sorry